model/clusters_mgmt/v1/billing_model_item_type.model

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright (c) 2023 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// BillingModelItem represents a billing model
+struct BillingModelItem {
```

Comment:
I wonder if this should be class? class have ID and HREF and Kind by default.

Reply:
Yes, it makes sense ... I copied it over from the accounts_mgmt folder, but I believe we better use the class in this case. 

I will updated it in both sides. cc: @tirthct

Reply:
@tzvatot I've replaced the struct it with a class. PTAL